### ISSUE TITLE
Variant idea: Double red (dark with no bdr)

### ISSUE THREAD

**Issue by ricardodd2** [Hanabi-Live/hanabi-live#2961]
I have got an idea for a variant that has dark difficutily but no bdrs on turn 1.

I thought of calling it double red (but it can be double rainbow, white, brown, pink etc.).
In practice the double suit is composed of 1111 222 333 444 55 and two stacks of that same suit have to be made.

**Comment by NZadeh** [comment]
Sounds fun, who's gonna code it up? :)

**Comment by ricardodd2** [comment]
> Sounds fun, who's gonna code it up? :)

If people approve, I could try when I have time. But I'd appreciate a little help from someone that knowns the code to start, since I never touched it

**Comment by newduke** [comment]
FWIW that's significantly easier than dark. "efficiency + pace" is not the same as "difficulty"

**Comment by ricardodd2** [comment]
That's true. It is easier enough that I though a 6s "triple double" is feasable. It would br a 1 off similar to mixes

**Comment by IceSelkie** [comment]
Similar to part of #2608

**Comment by ricardodd2** [comment]
Thx. It seemed too simple of an idea for no-one else to have thought about it. I think the dark version is the most interesting so I wouldn't bother adding the simple 6 4 4 4 2 and the double dark 2 2 2 2 2 versions.
